.github/workflows/debci.yml

```diff
@@ -29,6 +29,7 @@ jobs:
       # it's needed (will be auto-loaded) by routing.test_vrf_basic
       - name: Install dependencies
         run: |
+          sudo add-apt-repository -y -n -u -s ppa:slyon/netplan-ci
```

Comment:
Yes, exactly. This will be obsolete as soon as the `autopkgtest` package `5.37ubuntu3~24.04.1` migrates from `noble-proposed` to `noble-updates`. But the CI-PPA might still be useful for other stuff in the future..